src/types.ts

```diff
@@ -245,7 +271,7 @@ type ChartTypesFormAction = {
 }
 type TradersHubOnboardingFormAction = {
     action?: 'open' | 'close' | 'step_passed' | 'step_back' | 'choose_step_navigation'
-    form_source?: 'tradershub_dashboard_form' | 'tradershub_first_entrance' | 'repeat_tour'
+    form_source?: 'tradershub_dashboard_form' | 'tradershub_first_entrance' | '7_step'
```

Comment:
```suggestion
    form_source?: 'tradershub_dashboard_form' | 'tradershub_first_entrance' | 'repeat_tour'
```

`7_step` as a `form_source` was a mistake, and I changed it after discussing with Denis, please revert to `repeat_tour`